### PR TITLE
Adding asa-manager to the deployment process

### PR DESCRIPTION
**Pull request by ppathan**
# Type of change? 

- [x] Bug fix
- [x] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
* Enabling ASA manager deployment through basic and standard deployment
* Improving the setup vm script to accept parameters out of order.
* Fixing build break when using the latest typescript

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "@types/babel-generator": {
       "version": "6.25.1",
       "resolved": "https://registry.npmjs.org/@types/babel-generator/-/babel-generator-6.25.1.tgz",
-      "integrity": "sha512-nKNz9Ch4WP2TFZjQROhxqqS2SCk0OoDzGazJI6S+2sGgW9P7N4o3vluZAXFuPEnRqtz2A0vrrkK3tjQktxIlRw==",
+      "integrity": "sha1-+GqzzxMrBFl/5sQx0wg6rxt2tTA=",
```
Comment (jillcary) [inline package-lock.json]: Just curious if this should be sha512 instead.

File: solutions/remotemonitoring/scripts/all-in-one.yaml [hunk 2]
```diff
@@ -598,6 +598,112 @@ spec:
...
+apiVersion: v1
+kind: Service
+metadata:
+  name: asa-manager-svc
+  labels:
+    app: asa-manager
+spec:
+  type: NodePort
+  ports:
+  - port: 9024
```
Comment (jillcary) [inline solutions/remotemonitoring/scripts/all-in-one.yaml]: Do we have docs somewhere listing ports to services?

File: solutions/remotemonitoring/single-vm/setup.sh [hunk 3]
```diff
@@ -12,32 +12,42 @@ PKEY="${CERTS}/tls.key"
...
-export PCS_IOTHUBREACT_AZUREBLOB_KEY="${14}"
-export PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX="${15}"
-export PCS_CERTIFICATE="${16}"
-export PCS_CERTIFICATE_KEY="${17}"
-export PCS_AZUREMAPS_KEY="${18}"
-export PCS_AUTH_ISSUER="https://sts.windows.net/${5}/"
-export PCS_AUTH_AUDIENCE="$6"
+export HOST_NAME="localhost"
+export PCS_LOG_LEVEL="Info"
+export APP_RUNTIME="dotnet"
```
Comment (jillcary) [inline solutions/remotemonitoring/single-vm/setup.sh]: What happens if it's java? Does this get overwritten?

Comment (ppathan) [reply solutions/remotemonitoring/single-vm/setup.sh]: Yes it taken from --runtime

File: package-lock.json (hunk 1 above)
Comment (ppathan) [reply package-lock.json]: This file is auto-generated/updated by npm.

File: solutions/remotemonitoring/scripts/all-in-one.yaml (hunk 2 above)
Comment (ppathan) [reply solutions/remotemonitoring/scripts/all-in-one.yaml]: I couldn't find it @dluc do you know?

**Pull request merged**